src/senaite/core/profiles/default/workflows/senaite_sample_workflow/definition.xml

```diff
@@ -890,6 +897,7 @@
     <exit-transition transition_id="republish"/>
     <exit-transition transition_id="invalidate"/>
     <exit-transition transition_id="dispatch" />
+    <exit-transition transition_id="listing_multi_results"/>
```

Comment:
This transition should not be possible when the status is "published". At this point, results have been submitted, verified and published already.

Reply:
Yes agree. I thought about the usecase to just view all of the results at once for eventual comparison. But maybe the main use-case is results entry and not viewing the results.